api/db/models/petition.py

```diff
@@ -0,0 +1,31 @@
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Enum
+from sqlalchemy.orm import relationship
+from sqlalchemy.sql import func
+from ..base_class import Base
+from .supporter_petitions import supporter_petitions
+import enum
+
+
+class Status(enum.Enum):
+    active = "active"
+    victory = "victory"
+    closed = "closed"
+
+
+class Petition(Base):
+    __tablename__ = "petition"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String, nullable=False)
+    description = Column(String, nullable=False)
+    image = Column(String)
```

Comment:
Where are you planning to store images btw?

Reply:
For the start, I'm planning to save only the image URL here, but if I will have enough time I will try to connect something like an AWS S3 bucket and I will change this field to uuid